service/debugger/debugger_windows.go

```diff
@@ -14,3 +20,23 @@ func stopProcess(pid int) error {
 	// the process.
 	return nil
 }
+
+func verifyBinaryFormat(exePath string) error {
+	f, err := os.Open(exePath)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+
+	// Make sure the binary exists and is an executable file
+	if filepath.Base(exePath) == exePath {
+		if _, err := exec.LookPath(exePath); err != nil {
+			return err
+		}
+	}
+
+	if _, err = pe.NewFile(f); err != nil {
+		return proc.ErrNotExecutable
```

Comment:
Is this error ever returned anywhere else now? If not it should probably be moved here.

Reply:
No it's not... I think we can get rid of it and just return the `api.ErrNotExecutable` error instead.